src/components/shared/ConstructionPage.tsx

```
@@ -0,0 +1,107 @@
+            theme === 'dark'
+              ? 'bg-orange-500 hover:bg-orange-600 text-white'
+              : 'bg-orange-500 hover:bg-orange-600 text-white'
+          } transform hover:scale-105 shadow-lg hover:shadow-xl`}
+        >
+          <svg 
+            className="w-5 h-5 mr-2" 
+            fill="none" 
+            stroke="currentColor" 
+            viewBox="0 0 24 24"
+          >
+            <path 
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)